### PR TITLE
Fix User Re-Login on a back button

### LINKED ISSUE #331: User re-login after logging out and clicking back button.
## Description
As a user, when I log out and click the back button, I am able to re-login without entering any details, which is a flaw.

## Acceptance Criteria
Once logged out, the user mustn't re-login with a back button.

### Update [Required]
- [ ] Meet the acceptance criteria

### Enhancement to Update [Optional]
- [ ] Go through Portal, and check if there are any more such flaws. If they exist, create an issue.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by imrshu**
### Description
Django allauth built in Logout View has been extended into a custom logout view and cache control decorator has been added.

Fixes #331 

### Type of Change:
- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by imrshu** [comment]
@sa7i sorry I have forgot to run flake8 but now all tests have passed. So kindly review it :+1:

**Comment by sa7i** [comment]
@imrshu The views have to be added in common/views.py and tests in common/tests/test_views.py

**Comment by imrshu** [comment]
@sa7i View has been shifted in common. :+1:

**Comment by sa7i** [comment]
@imrshu Could you also add tests for the views?

**Comment by imrshu** [comment]
@sa7i Actually i would love to add a test view for this but i am a bit confuse as it will a user choice to press back button. So how could i write a view for that ?

**Comment by imrshu** [comment]
@sa7i it would be great if you could help me in this ?

**Comment by sa7i** [comment]
Refer this - https://stackoverflow.com/questions/29569676/django-tests-login-logout

**Comment by imrshu** [comment]
@sa7i Test case for login & logout have been added. kindly review them :+1:

**Comment by imrshu** [comment]
@sa7i Test case for login & logout have been added. Kindly review them :+1:

File: systers_portal/common/views.py [hunk 1]
```diff
@@ -15,3 +17,13 @@ class AboutUsView(TemplateView):
 
 class NewCommunityProposalView(TemplateView):
     template_name = "common/new_community_proposal.html"
+
+
+class Logout(LogoutView):
+
+    @cache_control(no_cache=True, must_revalidate=True, no_store=True)
```
Comment (tapasweni-pathak) [inline systers_portal/common/views.py]: after this we can remove `@login_required` present in any view.

**Pull request merged**